src/test/rustdoc/doc-cfg.rs

```diff
@@ -57,5 +57,7 @@ pub unsafe fn uses_target_feature() {
 //        'This is supported with target feature avx only.'
 #[doc(cfg(target_feature = "avx"))]
 pub fn uses_cfg_target_feature() {
-    uses_target_feature();
```

Comment:
Is this change an expected outcome?

Reply:
This change is not actually necessary, I noticed the `unsafe` error before removing the `analysis()` pass but now that pass is gone it will document without error. I undid the change.